python/paddle/fluid/tests/unittests/prim/prim/vjp/static/test_comp_div_grad.py

```diff
@@ -90,8 +90,8 @@ def train(self, use_prim, use_cinn):
 
     def test_cinn(self):
         paddle.disable_static()
-        dy_res = self.train(use_prim=False, use_cinn=False)
-        comp_st_cinn_res = self.train(use_prim=True, use_cinn=False)
+        dy_res = self.train(use_prim=False, use_cinn=True)
```

Comment:
Make this as False maybe, since this is baseline for test

Reply:
Done